### PR TITLE
fix for TOF calib

### PR DESCRIPTION
**Pull request by noferini**
Hi @chiarazampolli , @shahor02,
after checking the TOF calib output of the last cycle of relval I found that there is a problem when selecting FT0 BC when there are multiple-candidate: not always the correct one is taken because of a wrong condition (bug) in the code.
I didn't spot this problem in the previous LHC23zzh cpass* I checked, probably because the IR was lower.
Today I prepared a fix, tested on the debug production which gives the right results.
I apologize for not spotting the problem before (my fault).
Francesco

### CODE REVIEW

**Comment by chiarazampolli** [comment]
Hello @noferini ,

Thanks!! This is why we have tests 😀 
Once merged, i can patch the tag.

Chiara

**Comment by shahor02** [comment]
@noferini since the changes are trivial and you've tested them locally, I can merge it w/o waiting for the CI. Unless @chiarazampolli anyway will not tag it today. Let me know.

**Comment by chiarazampolli** [comment]
Hello @shahor02 ,

I will tag tomorrow early morning, letting the build ongoing while doing the CCDB uploads. 

For me you can merge, just to avoid further delays.

Chiara

**Review by shahor02 — APPROVED** [review]

**Comment by noferini** [comment]
Thanks!

**Pull request merged**
